playground/src/Editor/setupMonaco.tsx

```diff
@@ -620,6 +621,62 @@ function defineRustPythonAstLanguage(monaco: Monaco) {
   });
 }
 
+// Modeled after 'RustPythonAst'
+function defineCommentsLanguage(monaco: Monaco) {
```

Comment:
I tried looking into highlighting for ``code snippets``, but couldn't figure it out with a first-pass.